src/page/generate/ViewDocumentPage.tsx

```
@@ -746,6 +748,560 @@ Silakan lanjutkan pembuatan bagian berikutnya dari dokumen tanpa mengulangi kont
+            documentParagraphs.push(
+              new Paragraph({
+                text: element.title,
+                style: "Heading2",
+                spacing: { before: 240, after: 120 },
+                keepNext: true,
+              })
+            );
+          }
+          
+          // Handle markdown content
+          if (element.content && element.content.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)